src/utils/globalnav.ts

```diff
@@ -32,12 +32,12 @@ export const LEFT_NAV_LINKS = [
     url: 'https://ui.docs.amplify.aws/',
     order: 3
   },
-  // {
-  //   type: 'DEFAULT',
-  //   label: 'Contribute',
-  //   url: '/contribute/',
-  //   order: 4
-  // }
+  {
+    type: 'DEFAULT',
+    label: 'Contribute',
+    url: '/contribute/',
```

Comment:
This isn't shared between all the sites yet, but in sticking with the pattern do we want to use the full url here `https://docs.amplify.aws/contribute/` that way if we do happen to just copy and paste these changes to the other sites using the nav the link will still be correct?

Reply:
You're right, it should be consistent. Revised. Thanks!